mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -345,6 +341,31 @@ func resourceMongoDBAtlasAdvancedCluster() *schema.Resource {
 	}
 }
 
+var tagsSchema = schema.Schema{
+	Type:     schema.TypeSet,
+	Optional: true,
+	Elem: &schema.Resource{
```

Comment:
When comparing to `labels`, no `Set` function is defined as it is not needed. Using https://github.com/hashicorp/terraform/issues/10520#issuecomment-264681350 as reference, the default implementation will properly hash the entire data structure.